internal/printer/print-to-tsx.go

```diff
@@ -107,12 +111,12 @@ func renderTsx(p *printer, n *Node) {
 			// Without this, TypeScript can get tripped up by the body of our file.
 			if c.PrevSibling != nil && c.PrevSibling.Type == FrontmatterNode {
 				buf := strings.TrimSpace(string(p.output))
-				if len(buf) > 1 {
+				if len(buf)-len(getTSXPrefix()) > 1 {
 					char := rune(buf[len(buf)-1:][0])
 					// If the existing buffer ends with any character other than ;, we need to add a `;`
 					if char != ';' {
 						p.addNilSourceMapping()
-						p.print("\"\";")
+						p.print("{};")
```

Comment:
From my test, this seemed fine as a replacement of `"";`, but I feel like there must be a reason I went for `"";` back in the day. Can't remember it, though.

(When I first joined Astro, it was `;//`, from https://github.com/withastro/language-tools/pull/59 )